posthog/temporal/data_imports/sources/shopify/settings.py

```diff
@@ -1,19 +1,39 @@
-from posthog.temporal.data_imports.sources.shopify.constants import (
-    ABANDONED_CHECKOUTS,
-    ARTICLES,
-    SHOPIFY_PAYMENTS_BALANCE_TRANSACTIONS,
-)
-from posthog.warehouse.types import IncrementalField
+from dataclasses import dataclass
 
-INCREMENTAL_FIELDS: dict[str, list[IncrementalField]] = {
-    ABANDONED_CHECKOUTS: [
-        # {
-        #     "label": "created_at",
-        #     "type": IncrementalFieldType.DateTime,
-        #     "field": "created",
-        #     "field_type": IncrementalFieldType.Integer,
-        # }
-    ],
-    ARTICLES: [],
-    SHOPIFY_PAYMENTS_BALANCE_TRANSACTIONS: [],
+from posthog.temporal.data_imports.sources.shopify.constants import ABANDONED_CHECKOUTS, ARTICLES
+from posthog.warehouse.types import IncrementalField, IncrementalFieldType
+
+
+@dataclass
+class ShopifyIncrementalSetting:
+    fields: list[IncrementalField]
+    query_filter: str
+
+
+CREATED_AT = "created_at"
+UPDATED_AT = "updated_at"
+
+INCREMENTAL_SETTINGS: dict[str, ShopifyIncrementalSetting] = {
+    ABANDONED_CHECKOUTS: ShopifyIncrementalSetting(
+        fields=[
+            {
+                "label": "createdAt",
```

Comment:
The response object for [abandoned checkouts](https://shopify.dev/docs/api/admin-rest/latest/resources/abandoned-checkouts#get-checkouts?limit=1) has the created at timestamp be returned in snake case. We try to set the `label` to whatever the endpoint returns